Notebooks/Matching/fieldRotator.py

```diff
@@ -0,0 +1,107 @@
+from lsst.sims.utils import rotationMatrixFromVectors
+from lsst.sims.utils import cartesianFromSpherical, sphericalFromCartesian
+import numpy as np
```

Comment:
This is the new module I extracted the FieldRotator class into. I think it works better this way.